common/src/main/java/tachyon/heartbeat/HeartbeatThread.java

```diff
@@ -29,20 +32,27 @@
 
   private final String mThreadName;
   private final HeartbeatExecutor mExecutor;
-  private final long mFixedExecutionIntervalMs;
+  private HeartbeatTimer mTimer;
 
   /**
    * @param threadName Identifies the heartbeat thread name.
-   * @param hbExecutor Identifies the heartbeat thread executor; an
-   *        instance of a class that implements the HeartbeatExecutor
-   *        interface.
-   * @param fixedExecutionIntervalMs Sleep time between different heartbeat.
+   * @param executor Identifies the heartbeat thread executor; an instance of a class that
+   *        implements the HeartbeatExecutor interface.
+   * @param intervalMs Sleep time between different heartbeat.
    */
-  public HeartbeatThread(String threadName, HeartbeatExecutor hbExecutor,
-      long fixedExecutionIntervalMs) {
+  public HeartbeatThread(String threadName, HeartbeatExecutor executor, long intervalMs) {
```

Comment:
Actually, HeartbeatThread still uses thread name. I modified HeartbeatContext comments instead to use "executor thread name" instead of "executor name".